core/layout_api.php

```diff
@@ -996,7 +996,11 @@ function layout_page_content_end() {
  * @return null
  */
 function layout_breadcrumbs() {
-	echo '<div id="breadcrumbs" class="breadcrumbs">' , "\n";
+	$t_username = user_get_name( auth_get_current_user_id() );
+	$t_protected = current_user_get_field( 'protected' );
+	$t_access_level = get_enum_element( 'access_levels', current_user_get_access_level() );
```

Comment:
I (re)moved those three lines in https://github.com/mantisbt/mantisbt/commit/46fa6283d7364a1acd8b0176e127e245a8c2e374
Is there a reason you reintroduced them?